codespell_lib/_codespell.py

```diff
@@ -207,16 +207,15 @@ def init_chardet(self) -> None:
             raise ImportError(
                 "There's no chardet installed to import from. "
                 "Please, install it and check your PYTHONPATH "
-                "environment variable"
+                "environment variable",
```

Comment:
Why add a comma here?

Reply:
I am not an author of the PR and not intimate with black enough to know for sure if it is the one which did it, but I typically add such final commas in lists, tuples, etc (but not within jsons -- json does not tollerate them) to minimize possible subsequent diff since if another commit needs to add another item/option below, it would not need to change this line to add a comma. So I consider it nice convenience. And it seems black thinks so too.

Reply:
If the comma is needed by black, I will settle for it, although I don't like it after a single function argument.

Reply:
yeah, it came from black.